Add timeout error message to shadow reset

Wiljami opened the pull request:
The user is now informed when the ajax call times out.

When the ajax returns 504 timeout message the user now gets an error message about it.

Unfortunately there is no easy way to know if the shadow is actually resetted or not.

Related #315, #187

TODO: translations once code is ok

Wiljami commented:
Admittedly it is misleading. Should the message then say that the reset might or might not have occurred? As far as I know there is no way to check when the shadow has been reset last time.

elguitar commented:
Maybe the text could say that the reset is running in the background and that the user should check the shadow in a while. It should also say that if the user runs into trouble, she should find help from the docs and provide a link there.

Wiljami commented:
Changed the text and also changed the background color of the box to yellow to indicate that this is not outright error.

![Screenshot from 2020-08-03 14-20-44](https://user-images.githubusercontent.com/45056796/89177702-b7939500-d594-11ea-8144-4395281d91a1.png)

elguitar commented on modules/sitestatus.php:
```diff
@@ -410,6 +410,7 @@ public static function seravo_shadows_postbox() {
             </div>
           </div>
           <div class="alert" id="alert-failure"><button class="closebtn">&times;</button><p><?php _e('Failure!', 'seravo'); ?></p></div>
+          <div class="alert" id="alert-timeout"><button class="closebtn">&times;</button><p><?php _e('The shadow reset is still running on the background. You should check the status of the shadow after few minutes. If there are problems with the shadow, see the documentation from the link above.', 'seravo'); ?></p></div>
```
after **a** few

elguitar commented on languages/seravo-fi.po:
```diff
@@ -741,6 +741,16 @@ msgstr ""
+msgid "Bad plugins"
+msgstr "Tarpeettomat lisäosat"
+
+#: modules/plugin-checker.php:49
+msgid "Bad plugins found: "
+msgstr ""
+
```
Untranslated

elguitar replied:
But this has nothing to do with this PR :D Resolving
